api/src/main/java/org/openmrs/util/Security.java

```diff
@@ -70,6 +70,17 @@ public static boolean hashMatches(String hashedPassword, String passwordToHash)
 	}
 	
 	/**
+ 	 * This method is called when password cannot be encoded
+ 	 * @param algo algorithm used for encoding
+ 	 * @return the error message string with algorithm type used
+ 	 */
+ 	public static String passwordEncodeFail(String algo) {
+ 	  String errorMessage = "Can't encode password because the given algorithm: " + algo + " was not found! (fail)";
+ 		return errorMessage;
+ 	}
```

Comment:
@dkayiwa, @ivange94 My local builds are passing and I can provide screenshots if you want have no issue. So I'm a little bit confused on why the travis build is failing